Sanity/pkcs11/single-encrypted-disk/main.fmf

```diff
@@ -0,0 +1,36 @@
+summary: tests the clevis pkcs#11 luks functionality using a single encrypted disk
+component:
+  - clevis
+test: ./runtest-vm.sh
+recommend:
+  - clevis
+  - softhsm
+  - opensc
+  - jose
+  - cryptsetup
+  - openssl
+  - a2x
+  - git
+duration: 20m
+enabled: true
+tag:
+  - NoRHEL4
+  - NoRHEL5
+  - NoRHEL6
+  - NoRHEL7
+  - NoRHEL8
+  - TIPfail_Security
+  - Tier1
```

Comment:
Please remove Tier1 tags, needed special hw reqs not suitable for running in tiers.

Reply:
Removed from both the files